### ISSUE TITLE
UseStartsWithBackslash not detecting leading slash for functions/consts

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#229]
This is detected:
```
use \DateTimeImmutable;
```
This is not:
```
use function \strlen;
use const \DATE_ATOM;
```

CS version: 4.0.0

Also there is no autofix for this sniff. :/

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/baf23afb89b8ec1e19683d365357827aec094cae and implemented fixable in https://github.com/slevomat/coding-standard/commit/529a475d791a84a3f8eb297dbf2ca074467bedf3.

**Comment by Majkl578** [comment]
Thanks!
